shared/types/user.ts

```diff
@@ -29,6 +29,7 @@ export const UserBase = z.object({
       keyHash: z.string(),
       createdAt: z.date(),
       lastUsedAt: z.date().optional(),
+      isPlatform: z.string().optional(),
```

Comment:
Should `isPlatform` be a boolean?

Reply:
yes indeed! thanks for spotting it, fixed in [e043280](https://github.com/opengovsg/FormSG/pull/6608/commits/e043280fbb02947f38f6fdc96136d6778926677d)